lesson_02/8081/main.py

```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
+from flask import Flask, request
+import os
+import sys
+import requests
+import json
+
```

Comment:
remove it and add init.py to the folders

Reply:
I am not sure if I got it correctly, please check